app/jobs/bulkrax/create_relationships_job.rb

```diff
@@ -51,88 +49,75 @@ class CreateRelationshipsJob < ApplicationJob
     # parent_identifier or child_identifier param. For example, if a parent_identifier is passed, we know @base_entry
     # is the child in the relationship, and vice versa if a child_identifier is passed.
     def perform(parent_identifier:, importer_run_id:) # rubocop:disable Metrics/AbcSize
-      pending_relationships = Bulkrax::PendingRelationship.find_each.select do |rel|
-        rel.importer_run_id == importer_run_id && rel.parent_id == parent_identifier
-      end.sort_by(&:order)
-
-      @importer_run_id = importer_run_id
-      @parent_entry, @parent_record = find_record(parent_identifier, importer_run_id)
-      @child_records = { works: [], collections: [] }
-      pending_relationships.each do |rel|
-        raise ::StandardError, %("#{rel}" needs either a child or a parent to create a relationship) if rel.child_id.nil? || rel.parent_id.nil?
-        @child_entry, child_record = find_record(rel.child_id, importer_run_id)
-        if child_record
-          child_record.is_a?(::Collection) ? @child_records[:collections] << child_record : @child_records[:works] << child_record
-        end
+      importer_run = Bulkrax::ImporterRun.find(importer_run_id)
+      ability = Ability.new(importer_run.user)
```

Comment:
Bulkrax's CLI [doesn't currently require](https://github.com/samvera-labs/bulkrax/blob/main/lib/generators/bulkrax/templates/config/bulkrax_api.yml#L65-L66) `--user_id` to be set, so that could introduce unexpected errors here. 

I don't think the CLI currently gets a ton of use, so I don't think figuring out how to handle that is critical for this change to go through, but it's worth mentioning.

Reply:
That's definitely good to know.  And I suspect makes sense to have a fallback configured user.

Reply:
I can add that configuration, as it's good to capture that.